### DISCUSSION TITLE
ubuntu 20.04 python 3

### DISCUSSION THREAD

**Discussion by sojab0on** [prometheus/client_python#753] [General]
I used pycharm to interpet the commands you gave but it constantly says module notfound 

Is there a way import the module first 

I would like the client to run so my sabznbd exporter can work

**Comment by csmarchbanks** [comment]
Hello, have you installed the client e.g. `pip install prometheus-client`? If so, would you provide a code sample and/or more details about how you are encountering this error so I can help debug?

**Comment by sojab0on** [comment]
when i run the setup.py it give back this 
./setup.py
from: can't read /var/mail/os <-- mange to fix it by putting the files where it wants them 
from: can't read /var/mail/setuptools <-- mange to fix it by putting the files where it wants them 
./setup.py: regel 5: syntax error near unexpected sympbol '('
./setup.py: regel 5: `with open(path.join(path.abspath(path.dirname(__file__)), 'README.md')) as f:'
root@bitwarden:/home/michel/Documenten/client_python-0.12.0# pip install os

**Reply by csmarchbanks** [reply]
You should not need (or want) to run setup.py manually. Installing this library should be done via pip or another package manager.
